examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py

```diff
@@ -552,16 +558,20 @@ def transform_images(examples):
                     generator=generator,
                     batch_size=args.eval_batch_size,
                     output_type="numpy",
-                    num_inference_steps=args.ddpm_num_steps,
+                    num_inference_steps=args.ddpm_num_inference_steps,
                 ).images
 
                 # denormalize the images and save to tensorboard
                 images_processed = (images * 255).round().astype("uint8")
 
-                if args.logger == "tensorboard":
+                if args.logger == "tensorboard" and is_tensorboard_available():
                     accelerator.get_tracker("tensorboard").add_images(
                         "test_samples", images_processed.transpose(0, 3, 1, 2), epoch
                     )
+                elif args.logger == "wandb":
+                    accelerator.get_tracker("wandb").log(
+                        {"test_samples": [wandb.Image(img) for img in images_processed]}, step=global_step
+                    )
 
```

Comment:
Can we log the epoch instead of the step to keep it consistent with the tensorboard logging?